.angular-cli.json

```diff
@@ -22,6 +23,13 @@
             "styles": [
                 "styles.scss"
```

Comment:
if you include "sass/cashmere" here I don't think you will have to include this in all the other components